tools/c7n_mailer/c7n_mailer/handle.py

```diff
@@ -23,12 +23,13 @@
 from .sqs_queue_processor import MailerSqsQueueProcessor
 
 
-def config_setup(session):
+def config_setup(session, config=None):
     task_dir = os.environ.get('LAMBDA_TASK_ROOT')
     os.environ['PYTHONPATH'] = "%s:%s" % (task_dir, os.environ.get('PYTHONPATH', ''))
-    with open(os.path.join(task_dir, 'config.json')) as fh:
-        config = json.load(fh)
-    if config['ldap_bind_password']:
+    if not config:
```

Comment:
keep the capone code working for KMS decrypting the LDAP password, but allow an option to not do this. perhaps this should default to False instead of True, thoughts?